src/Components/AddItem/AddItem.js

```diff
@@ -0,0 +1,146 @@
+        setInputs({product: '', department: '', packeging: ''});
+
+        axios.post('http://localhost:5000/additem', inputsDetails)
+        .then(response => {
+            console.log(response.status);
+            if (response.data.inserted) {
+                setInserted(true)
+                setNotInserted(false)
+            }
+            else {
+                setInserted(false)
+                setNotInserted(true)
+            }})
+        .catch(error => {console.log(error, error.response)});
+    }
+
+    const alertInserted = () => {
+        return <div className="alertInserted">
+            <BsCheck2Circle className="icon" />
+            <div className="alert-text">
+                <span className="alert-title"> Success! </span> <br />
+                <span className='alert-added'> Item added successfully... </span> <br />
+            </div>
+            <AiOutlineClose className='close-alert-button-success' onClick={() => setInserted(false)}/>
+        </div>
+    }
+
+    const alertNotInserted = () => {
+        return <div className="alertNotInserted">
+            <FaExclamationCircle className="icon" />
+            <div className="alert-text">
+                <span className="alert-title"> Oops! </span> <br />
+                Something went wrong... <br />
+            </div>
+            <AiOutlineClose className='close-alert-button-oops' onClick={() => setNotInserted(false)}/>
+        </div>
+    }
+
+    return (
+        <>
+        <div className="add-item-block"> 
+            <h2>Add Item</h2>
+            <div className="add-item-explanation"> Add a new item to the database </div>
+            <form   className='addItem-form' 
+                    onSubmit={handleSubmit}>
+
+                <input  type="text"
+                        className="input-product"
+                        placeholder="Enter a new product..."
+                        name="product"
+                        value={inputs.product}
+                        onChange={handleChange} 
+                        /> <br />
+
+                <Select className="departments"
+                            value={selectedDepartment}
+                            placeholder="Choose a department..."
+                            onChange={setSelectedDepartment}
+                            options={departments} 
+                            name="department"/> <br />
+                
+                <Select className="input-packeging"
+                        value={selectedPackeging}
+                        placeholder="Choose a packeging type..."
+                        onChange={setSelectedPackaging}
+                        options={packeging} 
+                        name="packeging"/>
+                
+                <button className="add-item-button"> Add Item </button>
+            </form>
+        </div>
+
+        {inserted ? alertInserted() : null}
+        {notInserted ? alertNotInserted(): null}
```

Comment:
```suggestion
        {inserted ? alertInserted() : alertNotInserted()}
```

Reply:
by doing so, alertNotInserted() will pop up by default and it will say that something went wrong, until we will receive inserted: true from server